documents/design/AddNoiseToSimulation.md

```diff
@@ -0,0 +1,48 @@
+# Add Noise to Simulation
+
+*Issue #552*
+
+**Author:**
+- William Wilson
+
+## The Problem
+
+The problem I am trying to solve is the lack of noise in the gazebo simulation. In order to
+have a more realistic simulation and ground truth pose, noise is needed. In the real world,
+sensors output noisy data, which we must work through to get an accurate position of the
+robot. Currently, the simulation does not account for that. In the end, there will be noise
```

Comment:
Currently, the simulation actually does simulate noise. The problem is that our localization and mapping nodes are able to work with the current level of noise. The goal here is to create more of a "stress" test, ie.

If the localization node does a poor job and outputs a noisy / bad estimate, how does that affect the mapping stack? How does that affect the navigation stack?